waspc/src/Generator/FileDraft/WriteableMonad.hs

```diff
@@ -48,7 +49,20 @@ class (Monad m) => WriteableMonad m where
 
 instance WriteableMonad IO where
     createDirectoryIfMissing = System.Directory.createDirectoryIfMissing
-    copyFile = System.Directory.copyFile
+
+    -- NOTE(matija): this is the short version, not very nice?
+    --copyFile src dst = ((try (System.Directory.copyFile src dst)) :: IO (Either SomeException ())) >> return ()
+
+    copyFile src dst = do
+        -- NOTE(matija): we had cases (e.g. tmp Vim files) where a file initially existed
+        -- when the filedraft was created but then got deleted before actual copying was invoked.
+        -- That would make this function crash, so we just ignore those errors.
+        -- 
+        -- TODO(matija): we are currently fetching all the exceptions, which is not ideal. I tried IOException
+        -- but it wasn't the right exception type.
+        (_ :: Either SomeException a) <- try (System.Directory.copyFile src dst)
```

Comment:
Take a look at what we are doing in 
https://github.com/wasp-lang/wasp/blob/master/waspc/src/Util/IO.hs#L27
and 
https://github.com/wasp-lang/wasp/blob/master/waspc/src/Generator/Job/Process.hs#L86 .